### PR TITLE
add Stack Overflow as a possible support channel.

### PR DESCRIPTION
**Pull request by elharo**
@acozzette

### CODE REVIEW

**Review by elharo — COMMENTED** [review]
PTAL

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -9,8 +9,12 @@ assignees: ''
 
 <!--
 
-NOTE: this form is for bug reports only. For questions or troubleshooting, please see the protobuf mailing list: https://groups.google.com/forum/#!forum/protobuf
+NOTE: this form is for bug reports only.
+For questions or troubleshooting, please post on the protobuf mailing list:
+https://groups.google.com/forum/#!forum/protobuf or
+Stack Overflow https://stackoverflow.com/questions/tagged/protocol-buffers
 
+Google does formally monitor these channels, but there are many experienced developers there who also use protobuf.
```
Comment (acozzette) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: We do monitoring the mailing list, but not Stack Overflow.

**Review by elharo — COMMENTED** [review]
PTAL

**Review by acozzette — APPROVED** [review]

**Review by chill-otter — COMMENTED** [review]
``

**Pull request merged**